### ISSUE TITLE
Example does not work

### ISSUE THREAD

**Issue by soerenwolfers** [stephane-caron/qpsolvers#7]
Using exactly the example of the README gives

```
ValueError: constraints are inconsistent, no solution
```
This is with Python 3.6

**Comment by stephane-caron** [comment]
Strange... Can you do a ``pip list | grep quadprog`` and report the version number you gest?

You may also list your QP solvers:
```python
import qpsolvers
print(qpsolvers.available_solvers)
```
You can try other solvers from this list by adding the ``solver="solver_name"`` when calling the ``solve_qp`` function.

**Comment by stephane-caron** [comment]
I could reproduce the issue on a system that had qpsolvers 1.0.4 installed. The problem was solved in https://github.com/stephane-caron/qpsolvers/pull/5 released in qpsolvers 1.0.5, so all you have to do in this case is upgrade:
```bash
$ pip install --upgrade qpsolvers --user
```
(or ``sudo pip install --upgrade qpsolvers`` to install system-wide)
